$OpenBSD: patch-src_librustc_back_target_i686_unknown_openbsd_rs,v 1.5 2018/05/22 06:04:33 landry Exp $
i386 should be pentiumpro (no SSE2, no ...)
use lld by default for be able to deal with i128: https://github.com/rust-lang/rust/pull/50678/commits/0bef2402772218a20c7572e0d606e529844b527d
Index: src/librustc_back/target/i686_unknown_openbsd.rs
--- src/librustc_back/target/i686_unknown_openbsd.rs.orig
+++ src/librustc_back/target/i686_unknown_openbsd.rs
@@ -13,9 +13,10 @@ use target::{Target, TargetResult};
 
 pub fn target() -> TargetResult {
     let mut base = super::openbsd_base::opts();
-    base.cpu = "pentium4".to_string();
+    base.cpu = "pentiumpro".to_string();
     base.max_atomic_width = Some(64);
     base.pre_link_args.get_mut(&LinkerFlavor::Gcc).unwrap().push("-m32".to_string());
+    base.pre_link_args.get_mut(&LinkerFlavor::Gcc).unwrap().push("-fuse-ld=lld".to_string());
     base.stack_probes = true;
 
     Ok(Target {
